CraftTweaker2-API/src/main/java/crafttweaker/api/entity/IEntity.java

```diff
@@ -513,4 +513,9 @@ default boolean onGround() {
     default boolean isLightningBolt() {
         return false;
     }
+
+    @ZenGetter("UUID")
+    default String getUUID() {
```

Comment:
I believe with this, the IPlayer interface will have to be changed because it declares getUUID as well? Also UUID should preferably be renamed to uuid in the ZenGetter because consistence with camelCase and the player's uuid getter.

Reply:
> 
> 
> I believe with this, the IPlayer interface will have to be changed because it declares getUUID as well?

It should override it without any problems.